services/nodemailer/mail.ts

```diff
@@ -0,0 +1,27 @@
+// Nodemailer with SES transport, more info here: https://nodemailer.com/transports/ses/
+import { transporter } from "@nodemailer";
+
+/**
+ * Sends an email to the targetAddress email from the process.env.EMAIL_FROM email
```

Comment:
should this comment be changed because `process.env.EMAIL_FROM` is not relevant here?

Reply:
Fixed!